model-server/src/main/kotlin/org/modelix/model/server/handlers/HttpException.kt

```diff
@@ -116,3 +116,79 @@ class VersionNotFoundException(versionHash: String, cause: Throwable? = null) :
         type = "/problems/version-not-found",
         cause = cause,
     )
+
+/**
+ * An [HttpException] indicating that a provided repository name is not valid.
+ *
+ * @param invalidRepositoryId the invalid repository ID
+ * @param cause The causing exception for the bad request or null if none.
+ */
+class InvalidRepositoryId(invalidRepositoryId: String, cause: Throwable? = null) :
+    HttpException(
+        HttpStatusCode.BadRequest,
+        title = "Invalid repository ID.",
+        details = "Repository ID `$invalidRepositoryId` is not valid.",
+        type = "/problems/invalid-repository-id",
+        cause = cause,
+    )
+
+/**
+ * An [HttpException] indicating that an object key was uploaded without the corresponding value.
+ *
+ * @param objectKey the uploaded object key
+ */
+class UploadedObjectKeyWithoutObjectValueException(objectKey: String) :
```

Comment:
```suggestion
class ObjectKeyWithoutObjectValueException(objectKey: String) :
```

Wouldn't that be enough?